### PR TITLE
Fixes on vropsbuild, tested against a vROps 6.7 system

### PR DESCRIPTION
**Pull request by ssmiller25**
Address #62 for working build instructions on vROps itself.

### CODE REVIEW

File: vropsbuild.md [hunk 1]
```diff
@@ -4,19 +4,31 @@ In order to get this utility to work on a vROps system natively, a custom toolch
...
+```
 cd openssl/
 ./config --prefix=/usr --openssldir=/usr/local/openssl shared
 make
 make install
+echo "export LD_LIBRARY_PATH=/usr/local/lib${LD_LIBRARY_PATH}" >> ~/.bash_profile
+echo "export LC_ALL=en_US.utf8" >> ~/.bash_profile
+echo "export LANG=en_US.utf8" >> ~/.bash_profile
+# Reexecute base_profile, or relogin
+cd ~
```
Comment (jsirianni) [inline vropsbuild.md]: need to add: ```chmod +x ./.bash_profile```

Comment (ssmiller25) [reply vropsbuild.md]: Added fix

File: vropsbuild.md [hunk 2]
```diff
@@ -27,35 +36,49 @@ echo insecure >> $HOME/.curlrc
 wget https://raw.githubusercontent.com/curl/curl/curl-7_53_0/lib/mk-ca-bundle.pl
 perl mk-ca-bundle.pl -k
 export SSL_CERT_FILE=`pwd`/ca-bundle.crt
+echo "export SSL_CERT_FILE=`pwd`/ca-bundle.crt" >> ~/.bash_profile
+```
```
[the remark this answers is not in the document]
Comment (ssmiller25) [reply vropsbuild.md]: Done

File: vropsbuild.md [hunk 3]
```diff
@@ -27,35 +36,49 @@ echo insecure >> $HOME/.curlrc
...
+curl -O https://zlib.net/zlib-1.2.11.tar.gz
 tar -xzvf zlib-1.2.11.tar.gz
 cd zlib-1.2.11/
 ./configure
 make
 make install
+cd ..
 
-wget https://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz
+curl -O https://ftp.gnu.org/gnu/readline/readline-7.0.tar.gz
```
[the remark this answers is not in the document]
Comment (ssmiller25) [reply vropsbuild.md]: Added resourcing on line 24.

**Comment by jsirianni** [comment]
approved 👍

**Pull request merged**